src/main/java/ro/msg/learning/shop/configuration/LocationConfiguration.java

```diff
@@ -28,7 +38,20 @@ public LocationStrategy locationStrategy(@Value("${strategy.type}") String strat
             case "ABUNDANT" -> {
                 return new MostAbundantStrategy(stockRepository, stockMapper);
             }
+            case "GREEDY" ->
```

Comment:
Because you have now more values for the strategy type you can create an enum for them and use it here.